Libraries/LibJS/Lexer.cpp

```diff
@@ -205,8 +212,23 @@ Token Lexer::next()
     } else if (m_current_char == EOF) {
         token_type = TokenType::Eof;
     } else {
+        bool found_three_char_token = false;
+        if (m_position+1 < m_source.length()) {
+            char secondChar = m_source[m_position];
+            char thirdChar = m_source[m_position+1];
```

Comment:
`secondChar` should be `second_char` etc...
I think I forgot to change these in the original PR.